README.md

```diff
@@ -1,2 +1,133 @@
 # Enumerable-project
-In this project we learnt how to use a module inside a class. For this we created a class "MyList" and a module "MyEnumerable". The module "MyEnumerable" will implement a subset of the functionality of Enumerable.
+
+## Getting Started
```

Comment:
- [OPTIONAL] Kindly remove this line, as it is not required, you already have added it below after the `key-features` section.